apps/framework-cli/src/framework/python/wrappers/scripts/python_worker_wrapper/worker.py

```
@@ -15,29 +15,29 @@
-    """Collect all task names from DMv2 workflows, formatted as 'workflowName/taskName'.
+def collect_activities(workflows: dict[str, Workflow]) -> List[str]:
+    """Collect all task names from workflows, formatted as 'workflowName/taskName'.
 
     Args:
         workflows: Dictionary of workflow name to Workflow instance
 
     Returns:
         List[str]: List of activity names in format 'workflowName/taskName'
     """
-    log.info(f"<DMV2WF> Collecting tasks from dmv2 workflows")
+    log.info(f"<WF> Collecting tasks from workflows")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)